resources/qml/TimelineRow.qml

```diff
@@ -0,0 +1,122 @@
+import QtQuick 2.6
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.2
+
+import com.github.nheko 1.0
+
+import "./delegates"
+
+RowLayout {
+	property var view: chat
+
+	anchors.leftMargin: avatarSize + 4
+	anchors.left: parent.left
+	anchors.right: parent.right
+
+	height: Math.max(contentItem.height, 16)
+
+	Column {
+		Layout.fillWidth: true
+		Layout.alignment: Qt.AlignTop
+
+		//property var replyTo: model.replyTo
```

Comment:
Assuming this is commented out until I get around to fixing the reply functionality =D

Reply:
No, not really. What I actually wanted to do here is implement smart reply rendering, i.e. actually render the image in the reply by reusing the delegate and have a click on the reply scroll to the replay or so. That isn't that easy however, because I can't just instantiate the delegate with a different index it seems. So I need to create a json object or so for the replied-to message or so. No idea yet how I will actually implement it.